presto-native-execution/README.md

```diff
@@ -27,7 +27,15 @@ available inside `presto/presto-native-execution/scripts`.
 Create a directory say `dependencies` and invoke one of these scripts from
 this folder. All the dependencies are installed in the system default location eg: `/usr/local`.
 
+On an intel macOS machine, you can install the dependencies using the following commands:
+```bash
+./scripts/setup-macos.sh
+```
 
+On an M1 macOS machine, you can install the dependencies using the following commands:
+```bash
+CPU_TARGET="arm64" INSTALL_PREFIX="/opt/homebrew" ./scripts/setup-macos.sh
```

Comment:
We need to and can automate this. I will add a fix and we can avoid this update.